pkg/errors/cdc_errors.go

```diff
@@ -417,6 +413,10 @@ var (
 		"craft codec invalid data",
 		errors.RFCCodeText("CDC:ErrCraftCodecInvalidData"),
 	)
+	ErrCodecRowTooLarge = errors.Normalize(
```

Comment:
How about rename it to `ErrMessageTooLarge`

Reply:
Done